docs/advanced/configure-cf-external-databases-using-azure-mysql-postgres-service/migrate-mysql-databases/README.md

```diff
@@ -0,0 +1,97 @@
+# Migrate Cloud Foundry MySQL databases to Azure Database for MySQL
+
+On Cloud Foundry (CF), it is recommended to use PAAS service for the purpose of high availability. [Azure Database for MySQL](https://azure.microsoft.com/en-us/services/mysql/) is one of services that CF can levege for HA. 
+
+For a new deployment, you can refer to this [guidance](../README.md) to use Azure Database for MySQL as external databases.
+For an existed deployment, if you want to use Azure Database for MySQL, this document will provide a simple guildance to migrate the database.
+
+**NOTE:** Be careful on operating databases, you should practise on a test environment before officially do the migration on your production environment.
+
+In this guidance, we will use `mysqldump` and `mysql` command-lines to dump backup files from internal databases and then retore them to Azure Databases for MySQL.
+
+1. Figure out databases to be migrated
+
+    Check your CF manifest carefully, find out which databases are used. Write down database name, user, password for  step `Migrate databases`. If you are using Ops Manager, follow this [guid](https://docs.pivotal.io/pivotalcf/2-2/customizing/credentials.html) to get credentials.
```

Comment:
extra white space in "password for  step"

Reply:
good catch, will change.